app/models/notifier.rb

```diff
@@ -168,12 +170,24 @@ def changeset_comment_notification(comment, recipient)
                   I18n.t("notifier.changeset_comment_notification.commented.subject_other", :commenter => @commenter)
                 end
 
+      attachments.inline["logo.png"] = File.read(Rails.root.join("app", "assets", "images", "osm_logo_30.png"))
```

Comment:
I've changed it to `File.read("#{Rails.root}/app/assets/images/osm_logo_30.png")`, as discussed below.